tests/utils/test_tarfile.py

```diff
@@ -31,30 +31,3 @@ def test_not_secure_path():
         TarInfo("/bla/blu/ble"),
     ]
     assert [] == list(secure_path(test_list))
-
-
-def test_exclude_filter_good():
-    """Test exclude filter."""
-    filter_funct = exclude_filter(["not/match", "/dev/xy"])
-    test_list = [
-        TarInfo("test.txt"),
-        TarInfo("data/xy.blob"),
-        TarInfo("bla/blu/ble"),
-        TarInfo("data/../xy.blob"),
-    ]
-
-    assert test_list == [filter_funct(result) for result in test_list]
-
-
-def test_exclude_filter_bad():
-    """Test exclude filter."""
-    filter_funct = exclude_filter(["*.txt", "data/*", "bla/blu/ble"])
-    test_list = [
-        TarInfo("test.txt"),
-        TarInfo("data/xy.blob"),
-        TarInfo("bla/blu/ble"),
-        TarInfo("data/test_files/kk.txt"),
-    ]
-
-    for info in [filter_funct(result) for result in test_list]:
-        assert info is None
```

Comment:
Don't remove tests, fix it

Reply:
I removed the test as the function is not used anymore. If you prefer having dead code in your codebase, I can revert that...

Reply:
Make sure that you function are covered by tests. You remove the filter tests without add a new test for you new code. We can not merge PR they lost test coverage.

Reply:
I've implemented the same test cases for the new `_is_excluded_by_filter` function.